FrenchFilter/sections/adservers.txt

```diff
@@ -2,5 +2,6 @@
 ! Section contains list of advertising networks
 !
 ||mystighty.info^
+||i4tvy8vk9f74.com^

```

Comment:
why without `$third-party`?

Reply:
Also popup opens when click on player.

Reply:
> why without `$third-party`?

because first party requests to this domain don't exist, so we can block all requests

Reply:
> Also popup opens when click on player.

I will check it

Reply:
> because first party requests to this domain don't exist, so we can block all requests

in this case this is impossible to distinguish the rules for blocking third-party script and blocking popup in Safari(page content completely).

Reply:
ok, i will add $third-party

Reply:
> Also popup opens when click on player.

I can't reproduce the popup, it blocked by `||hdslecteur.xyz/asset/sostress.js` rule